### PR TITLE
Add participants list to cohort review

### PR DESCRIPTION
**Pull request by tjennison-google**
* Refactor common review requests into custom hooks.
* Refactor entity processing to also handle keys for review instances.

### CODE REVIEW

**Comment by tjennison-google** [comment]
image

I haven't included any config changes in this PR. I'll bundle them in with the changes to add the remaining cohort review domains to avoid having to expand all of the underlays twice.

**Review by marikomedlock — APPROVED** [review]
What new UI config will this use? I thought maybe the (demographic) attributes to show in the columns. And thanks for the screenshot, very helpful!

**Comment by tjennison-google** [comment]
> What new UI config will this use? I thought maybe the (demographic) attributes to show in the columns. And thanks for the screenshot, very helpful!

The config that's added in `underlaysSlice.ts`. It's attributes on person but you don't necessarily want to display them all. It also needs to add column widths and titles to the attributes names.

File: ui/src/data/source.tsx [hunk 1]
```diff
@@ -1245,22 +1245,20 @@ function processEntitiesResponse(
   return nodes;
 }
 
-function processEntityInstance(
-  attributeID: string,
-  instance: tanagra.InstanceV2
+function makeDataEntry(
+  primaryKey: string,
```
[the remark this answers is not in the document]
Comment (tjennison-google) [reply ui/src/data/source.tsx:1249]: This function is used on any entity, so it's the key for whatever entity is being processed. It's the primary entity instance id in the review case.

**Pull request merged**